src/CppUTestExt/MockSupport_c.cpp

```diff
@@ -32,6 +32,16 @@
 #include "CppUTestExt/MockSupport.h"
 #include "CppUTestExt/MockSupport_c.h"
 
+extern "C"
+{
+    typedef void (*cpputest_c_function_pointer)();
+}
+
```

Comment:
I'd prefer not as this is more descriptive and your editor should autocomplete anyway :)